facility-manager.html

```diff
@@ -0,0 +1,806 @@
+</div> <!-- end #line-block -->
+
+<h2><a class="anchor" id="imagej"></a>ImageJ and Fiji</h2>
+
+<p>Using the OMERO.insight plugin, ImageJ and Fiji can be used to open and view images on the OMERO server, draw ROIs and save the ROIs back to the OMERO server. Results of analysis performed in ImageJ or Fiji can be saved back to the OMERO server, attached as a file to the associated image. Full details of how to install the OMERO.insight plugin, and the workflows for using ImageJ and Fiji with OMERO are in the <a href="http://help.openmicroscopy.org/imagej.html" target="_blank" class="external-url">Using ImageJ with OMERO</a> section of the User Help.</p>
+
+<p class="top_link"><a href="#top"><img src="images/back_to_top.png" alt="Image for link to top of page" style="float: right;"/></a></p>
+
+<div class="line-block">
+<div class="line"><br /></div> <!-- end #line -->
+</div> <!-- end #line-block -->
+
+<h2><a class="anchor" id="disk"></a>Disk usage</h2>
+
+<p>System administrators can view disk usage using the OMERO.web Admin tab. The disk usage reported reflects only data owned by the users or groups. Files that are uploaded to the server, or created in situ, and attached to Projects, Datasets or Images, are not included in the disk usage statistics.</p>
+
+<ol>
+
+<li>
+<p class="first">Click on the <span class="bold">Statistics</span> tab to see the disk usage for all users and groups on the OMERO server.</p>
+
+<div class="line-block">
+<div class="line"><br /></div> <!-- end #line -->
+</div> <!-- end #line-block -->
+
+<img alt="../images/facilityManagerStats.jpg" class="align-center" src="images/facilityManagerStats.jpg" style="width: 100%;" />
+
+<div class="line-block">
+<div class="line"><br /></div> <!-- end #line -->
+</div> <!-- end #line-block -->
+
+</li>
+
+</ol>
+
+<p class="top_link"><a href="#top"><img src="images/back_to_top.png" alt="Image for link to top of page" style="float: right;"/></a></p>
+
+<div class="line-block">
+<div class="line"><br /></div> <!-- end #line -->
+</div> <!-- end #line-block -->
+
+<h2><a class="anchor" id="email"></a>Emailing users</h2>
+
+<p>The OMERO server can be configured so that administrators can use the OMERO.web Admin screen to email users through the OMERO system. Details on the configuration properties for mail are on <a href="https://www.openmicroscopy.org/site/support/omero5.2/sysadmins/config.html#mail" target="_blank" class="external-url">Configuration properties glossary</a> in the developer documentation.</p>
+
+<p>Recipients can be selected from all users, specific users or specific groups.</p>
+
+<ol>
+
+<li>
+<p class="first">Click on the <span class="bold">Email</span> tab to send emails to OMERO users or groups.</p>
+
+<div class="line-block">
+<div class="line"><br /></div> <!-- end #line -->
+</div> <!-- end #line-block -->
+
+<img alt="../images/sharingDataEmail.jpg" class="align-center" src="images/sharingDataEmail.jpg" style="width: 100%;" />
```

Comment:
Pass - Will or Ola will have to change it. 

An inactive user is what you do to one because you can't delete them.